<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CloudSetDlg</class>
 <widget class="QDialog" name="CloudSetDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>467</width>
    <height>349</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>33</x>
     <y>60</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>公司域名：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="domainEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>260</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="domainBtn">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>60</y>
     <width>55</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>提交</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>33</x>
     <y>110</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>公司名称：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>110</y>
     <width>260</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="nameBtn">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>110</y>
     <width>55</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>提交</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>33</x>
     <y>160</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>IDC名称：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="idcEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>160</y>
     <width>260</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="idcBtn">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>160</y>
     <width>55</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>提交</string>
   </property>
  </widget>
  <widget class="QPushButton" name="moduleBtn">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>210</y>
     <width>55</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>提交</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="moduleEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>210</y>
     <width>260</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>33</x>
     <y>210</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>模块名：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>参数设置</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="saveBtn">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>280</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="quitBtn">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>280</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>quitBtn</sender>
   <signal>clicked()</signal>
   <receiver>CloudSetDlg</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>294</y>
    </hint>
    <hint type="destinationlabel">
     <x>233</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
